python_files/generate_supplementary_plots.py

```diff
@@ -25,6 +25,29 @@
 
 
 # Panel validation
+panel_validation_viz_dir = os.path.join(SUPPLEMENTARY_FIG_DIR, "panel_validation")
+if not os.path.exists(panel_validation_viz_dir):
+    os.makedirs(panel_validation_viz_dir)
+
+controls_dir = "/Volumes/Shared/Noah Greenwald/TONIC_Cohort/image_data/controls"
+controls_fov = "TONIC_TMA1_colon_bottom"
+supplementary_plot_helpers.validate_panel(
+    controls_dir, controls_fov, panel_validation_viz_dir, font_size=180
+)
+
+samples_dir = "/Volumes/Shared/Noah Greenwald/TONIC_Cohort/image_data/samples"
+samples_fov = "TONIC_TMA2_R5C4"
+samples_channels = sorted(io_utils.remove_file_extensions(
+    io_utils.list_files(os.path.join(samples_dir, samples_fov), substrs=".tiff")
+))
+exclude_chans = ["CD11c_nuc_exclude", "CK17_smoothed", "ECAD_smoothed", "FOXP3_nuc_include",
```

Comment:
Doesn't look like this is working, the stitched image still has the `chan` ones. Also exclude noodle, gold, etc

Reply:
Oops, looks like a mistake when deleting and renaming the files. I just regenerated.